deploy/template.yaml

```diff
@@ -187,9 +187,12 @@ Resources:
       Environment:
         Variables:
           CREDENTIAL_ISSUERS_CONFIG_PARAM_PREFIX: !Sub "/${Environment}/credentialIssuers/ukPassport/clients"
+          CRI_PASSPORT_SESSIONS_TABLE_NAME: !Select [1, !Split ['/', !GetAtt CriPassportSessionsTable.Arn]]
```

Comment:
You should be able to use simply `!Ref CriPassportSessionsTable` as described here: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-dynamodb-table.html#aws-resource-dynamodb-table-return-values
>For the resource with the logical ID myDynamoDBTable, Ref will return the DynamoDB table name.